remote_engine_client/src/client.rs

```diff
@@ -66,7 +81,7 @@ impl Client {
             Err(e) => {
                 // If occurred error, we simply evict the corresponding channel now.
                 // TODO: evict according to the type of error.
-                self.cached_router.evict(&table_ident).await;
+                self.cached_router.evict(&[table_ident]).await;
```

Comment:
Frequent eviction may lead to poor performance, so how about leaving a log for it?